src/components/Meme.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react'
+
+export default function Meme() {
+    const [meme, setMeme] = useState({
+        topText: "",
+        bottomText: "",
+        randomImage: "https://i.imgflip.com/39t1o.jpg"
+    })
+
+    const [allMemes, setAllMemes] = useState([]);
+
+    useEffect(() => {
+        async function getMemes() {
+            const response = await fetch("https://api.imgflip.com/get_memes")
+            const data = await response.json()
+            setAllMemes(data.data.memes)
```

Comment:
To avoid the repetition of data, I can suggest improving your code  by using  destructuring Method
```suggestion
            const {data} = await response.json()
            setAllMemes(data.memes)
```